### DISCUSSION TITLE
Project: Thư viện beer và lựa chọn phù hợp với từng người dùng

### DISCUSSION THREAD

**Discussion by ntdo2506** [OngDev/community#3] [Project Ideas]
1. Thành phần - Nguồn gốc Beer
2. Nồng độ cồn
3. Mùi vị - Màu sắc - Giới tính
4. Check độ phù hợp với beer: Yêu màu hồng sẽ uống bia gì, sống nội tâm sẽ phù hợp bia gì
v..v mây mây ra

**Comment by duynguyenod** [comment]
Đink kout man :v

**Reply by ntdo2506** [reply]
Vote dùng Vue với Django nha!

**Reply by quanghuy1288** [reply]
đã code chưa các thánh :)

**Reply by cutbypham** [reply]
> đã code chưa các thánh :)

@quanghuy1288
Chắc xong cái project course manager mới sử lý em cái ideas này ý, sư huynh. (hên sui, tùy theo viewer kênh ông dev vote)
https://github.com/OngDev/course-manager/
